ui-admin/src/study/participants/participantList/ParticipantList.tsx

```diff
@@ -75,7 +76,7 @@ function ParticipantList({ studyEnvContext }: {studyEnvContext: StudyEnvContextT
           </Button>
         </div>
       }
-
+      <div><Link to={`withdrawn`}>Withdrawn</Link></div>
```

Comment:
should we put this under a superuser only flag until we figure out the right place for it?

Reply:
this feature was explicitly requested by OurHealth -- they want to be able to see the list of withdrawn participants